lib/vehicle.rb

```diff
@@ -5,14 +5,19 @@ class Vehicle
               :year,
               :make,
               :model,
-              :engine
+              :engine,
+              :registration_date
+              
+  attr_accessor :plate_type
```

Comment:
We do not want an attr_accessor for plate_type since you created a method called `set_plate_type` that will do this for us. I like the helper method better than having an attr_accessor.